TrackParamFilter/sections/specific.txt

```diff
@@ -9,6 +9,8 @@
 ! https://github.com/AdguardTeam/AdguardFilters/issues/136734
 ||investing.com^$cookie=smd
 ||investing.com^$cookie=udid
+! https://github.com/AdguardTeam/AdguardFilters/pull/137576
+||basketballking.jp^$removeparam=cx_art
```

Comment:
@Alex-302 @zloyden IMO this should be generic and added to `! Cxense clickthrough tracking` line.

Reply:
@Yuki2718 is it related to Cxense?
I see it on 4 sites only. 3 - with no traffic.

Reply:
> is it related to Cxense?

Supposedly. Let's see if it's used in more sites.